modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/components/ViolationPopover.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './ViolationPopover.scss';
+
+import ClayIcon from '@clayui/icon';
+import ClayList from '@clayui/list';
+import ClayPopover from '@clayui/popover';
+import React, {useCallback, useMemo, useState} from 'react';
+
+import {useObserveRect} from '../hooks/useObserveRect';
+
+import type {RuleRaw} from '../hooks/useA11y';
+
+const Overlay = React.forwardRef<
+	HTMLDivElement,
+	React.ButtonHTMLAttributes<HTMLDivElement>
+>(({style, ...othersProps}, ref) => (
+	<div {...othersProps} className="a11y-overlay" ref={ref} style={style}>
```

Comment:
Since you aren't adding anything to `style` here, can you just pass all props through?

Reply:
Yeah we don't actually need `othersProps`, this is just the mental strength when we build stuff in Clay 😂